build.sbt

```diff
@@ -251,7 +267,7 @@ lazy val commonScalacOptions = Seq(
   "-language:implicitConversions",
   "-language:experimental.macros",
   "-unchecked",
-  "-Xfatal-warnings",
+  //"-Xfatal-warnings",
```

Comment:
Yes, this is necessary for now for `cats-macros` due to differences between 2.10 and 2.11. I think I can fix this though so I'll re-enable it when I do.